main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
```

Comment:
A quick note: in Python // isn't valid for comments. Replace it with #:
```suggestion
/# Loop back to this point once code finishes
```